content/en/docs/ambient/overview/APIs/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Ambient APIs
+description: Understanding ambient APIs and its status.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This page lists all the user facing APIs relevant to the ambient mode and its feature status.
+
+## Labels
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `istio.io/dataplane-mode` | Beta |
+| `istio.io/use-waypoint` | Beta |
+| `istio.io/use-waypoint-namespace` | Beta |
+| `istio.io/waypoint-for` | Beta |
+
+TODO: add link to labels page for each when 3307 is merged.
+
+## Annotations
+
+Below are annotations that can be used to customize your waypoint proxies:
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `gateway.istio.io/service-account` | Alpha |
```

Comment:
Why are these annotations and not labels? IIRC, @bleggett was pushing for a convention of annotations == controller-set and labels == user-intent and I agree.

Reply:
PR: https://github.com/istio/istio/pull/43103/files predates that.

I'll sign up to move them to labels if there's agreement we should.

Reply:
I am not super convinced; there are many many label and annotations that do not align with this, so we would need to change 100s

Reply:
I don't think there are 100s in ambient specifically.

That being said - this _should_ probably be something we explicitly address before moving an annotation/label from `Alpha`

Reply:
`gateway.istio.io/service-account` isn't really ambient, it existed before ambient and impacts GW in general. A large portion of annotations probably apply to sidecar+ambient

Reply:
Can we duplicate the ambient annotations into labels and gradually deprecate the annotations?

Reply:
Or, alternatively, use a completely different label for ambient (though this would result in more confusion IMO)

Reply:
> `gateway.istio.io/service-account` isn't really ambient, it existed before ambient and impacts GW in general. A large portion of annotations probably apply to sidecar+ambient

Fair enough - but if it's Alpha, its ambient-ness kinda doesn't matter. It shouldn't graduate to beta or beyond and be documented until we make it what it should be.

Reply:
> Or are they already stable?

already stable. so ship has sailed on these. In general though I _would_ strongly suggest we don't promote labels or annos out of Alpha until we make sure they're in the right bucket, going forward.

That seems like a pretty minimally reasonable quality gate and just because we haven't been consistent in the past doesn't mean we should avoid starting now.